torch/csrc/jit/netdef_converter.h

```diff
@@ -11,7 +11,7 @@ namespace jit {
  * The NetDef \p net is converted and the result is stored in the
  * torch::jit::Graph \p graph. The function also records name->value map in \p
  * valueMapPtr. If the original net had several values with the same name, the
- * map will contain the value for the last definition.
+ * map will contain the value for the last definition. valueMapPtr is optional.
```

Comment:
Maybe we should then add a `nullptr` default value for `valueMapPtr` argument then?